docs/content/topics/operators/gateways.md

```diff
@@ -11,340 +11,518 @@ aliases:
+is an entity that generates events. Often times, it translates some external
+trigger into a Brigade event.
 
-Brigade ships with the ability to enable various gateways that are ready to go.
-
-These include the [Container Registry Gateway](dockerhub.md), the [Github Gateway](./github.md)
-and the [Generic Gateway](./genericgateway.md).  They can all be enabled via top-level
-Helm [chart flags](https://github.com/brigadecore/charts/blob/master/charts/brigade/values.yaml).
+While there is no default gateway that ships with Brigade, installing one
+alongside Brigade is as simple as a [Helm] chart install, along with any
+additional setup particular to a given gateway. See
+[below](#available-gateways) for a current listing of v2-compatible gateways.
 
 All of these provide HTTP-based listeners that receive incoming requests
-(from a container registry, Github or other platforms and systems) and generate
-Brigade events as a result.
+(from Github or other platforms and systems) and generate Brigade events as a
+result.
 
 However, Brigade's gateway system works with more than just webhooks.
 
-For example, the `brig` client also acts as a gateway. When you execute a `brig run`
-command, `brig` creates a Brigade event. By default, it emits an `exec` event. And
-Brigade itself processes this event no differently than it processes the GitHub
-or container registry events.
+For example, the `brig` client also acts as a gateway. When you execute a
+`brig event create` command, `brig` creates a Brigade event of source
+`brigade.sh/cli` and of type `exec`. Brigade itself processes this event no
+differently than it processes events from gateways tracking external activity.
 
 There are no rules about what can be used as a trigger for an event. One could
-write a gateway that listens on a message queue, or runs as a chat bot, or watches
-files on a filesystem... any of these could be used to trigger a new Brigade event.
+write a gateway that listens on a message queue, or runs as a chat bot, or
+watches files on a filesystem... any of these could be used to trigger a new
+Brigade event.
+
+The remainder of this guide explains how gateways work and how you can create
+custom gateways.
+
+[Helm]: https://helm.sh
 
-The remainder of this guide explains how gateways work and how you can create custom
-gateways.
+## Available Gateways
 
-## An Event Is A Secret
+Currently, the list of official Brigade v2 gateways that process external
+events is as follows:
 
-The most important thing to understand about a Brigade event is that it is simply
-a [Kubernetes Secret](https://kubernetes.io/docs/concepts/configuration/secret/)
-with special labels and data.
+* [Github Gateway](https://github.com/brigadecore/brigade-github-gateway)
+* [BitBucket Gateway](https://github.com/brigadecore/brigade-bitbucket-gateway/tree/v2)
+* [CloudEvents Gateway](https://github.com/brigadecore/brigade-cloudevents-gateway)
+* Noisy Neighbor?
```

Comment:
See https://github.com/brigadecore/brigade/pull/1569/files#r690545417.

I think I would remove this from the list.

I've been counting it in a separate category.